docs/pages/access-controls/guides/guide-to-cluster-wide-idp-hardening.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Guide to hardening against IdP compromises
+description: Implement cluster-wide hardening measures.
+---
+
+Protecting your organization against Identity Provider (IdP) compromises is more critical than ever. 
+This guide aims to help you fortify your identity infrastructure and mitigate the risks associated with IdP weaknesses.
+
+## Understanding IdP compromise
+
+An IdP compromise occurs when an attacker gains unauthorized access to your identity management 
+system, potentially allowing them to impersonate legitimate users, escalate privileges, or access 
+sensitive information. This can happen through various means, such as exploiting software vulnerabilities, 
+stealing credentials, or social engineering attacks.
+
```

Comment:
Let's include this diagram:

<img width="938" alt="Screenshot 2024-06-27 at 8 10 48 PM" src="https://github.com/gravitational/teleport/assets/13717/fa63720f-8c6d-4e14-a571-24bcc6545cdf">

I have a vector version as well if you need it.